src/azimuth/view/hud.c

```diff
@@ -700,7 +700,7 @@ static void draw_upgrade_box_message(
     az_upgrade_t upgrade, double height, int gun_instructions_top,
     az_clock_t clock) {
   const char *name = az_upgrade_name(upgrade);
-  const char *description = az_upgrade_description(upgrade, &player->upgrades);
+  const char *description = az_upgrade_description(upgrade, &player->upgrades, prefs);
```

Comment:
good point, thanks for the catch.  i propose just using the overload here (if specified, otherwise the default), like i suggested earlier for rocket/bomb upgrades.

here's essentially using [$k], with some default values:
https://pasteboard.co/IR6seYY.png

here's with all overrides with the longest possible keynames:
https://pasteboard.co/IR6sHtW.png

Reply:
Ditto, looks good!